### DISCUSSION TITLE
localizedText question

### DISCUSSION THREAD

**Discussion by AllenColbert** [eclipse-milo/milo#1153] [General]
Hi kavin,I referred to the code in `org.eclipse.milo.examples.client.EventSubscriptionExample` and wrote a subscription alarm&event function. and expected to receive alarm Message in multiple languages, but I can only receive it in English now. 
Here is my question:
1. I have checked the alarm information using UaExpert and it can correctly display the language I have set up.Can this prove that there are no issues with my OpcUa server？
2. Is there a way for me to obtain message for the language I have set up？

Thanks @kevinherron

**Comment by kevinherron** [comment]
What are you doing to request a different locale from the server?
